console/v1/register.go

```diff
@@ -32,6 +32,7 @@ func Resource(resource string) schema.GroupResource {
 func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(GroupVersion,
 		&ConsoleLink{},
+		&ConsoleLinkList{},
```

Comment:
If you are going to add this one, want to add the `*List{}` for the rest as well?

Reply:
You are correct in that it should be here, see:
https://github.com/openshift/api/blob/master/config/v1/register.go#L43

Reply:
> If you are going to add this one, want to add the *List{} for the rest as well?

Nice catch! Added lists for `ConsoleCLIDownload` and `ConsoleNotification` but not for `ConsoleExternalLogLink` since such type does not exist.
I assume that it's normal, e.g. BuildLog does not have List type as well https://github.com/openshift/api/blob/master/build/v1/register.go#L39

Reply:
They should all 3 have a list type. 
Opened #428, thanks for noticing that!

Reply:
I would say we should merge in #428 first, or just update your PR to include and I will close #428.

Reply:
Let me know if you want to just import that, it is only a few lines.

Reply:
@benjaminapetersen 

Rebased against master.
Cherry-picked your commit with ConsoleExternalLogLinkList type
Generate deepcopy and swagger doc for ConsoleExternalLogLinkList
Register ConsoleExternalLogLinkList as well as other List types.

So, please close your PR